docs/dagster-university/pages/dagster-essentials/lesson-1/orchestration-approaches.md

```diff
@@ -10,8 +10,9 @@ But first, what’s a workflow?
 
 {% callout %}
 
-> A workflow is a system for managing repetitive processes and tasks which occur in a particular order. They are the mechanism by which people and enterprises accomplish their work \[…].\_ - [IBM](https://www.ibm.com/topics/workflow)
-> {% /callout %}
+> A workflow is a system for managing repetitive processes and tasks which occur in a particular order. They are the mechanism by which people and enterprises accomplish their work \[…].\_ - [IBM](https://www.ibm.com/topics/workflow) >
+
+{% /callout %}
```

Comment:
this was an oversight from previous `yarn md-transform` commit - when using quote `> `, we need to make sure there's a new line after the quoted content; otherwise, the linter would treat the following paragraph as part of the quotes